settings.py

```diff
@@ -138,9 +138,9 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.common.BrokenLinkEmailsMiddleware',
     'corehq.middleware.OpenRosaMiddleware',
-    'corehq.middleware.TimeoutMiddleware',
     'corehq.util.global_request.middleware.GlobalRequestMiddleware',
     'corehq.apps.users.middleware.UsersMiddleware',
+    'corehq.middleware.TimeoutMiddleware',
```

Comment:
changed the middleware order because the UsersMiddleware is what creates the request.domain attribute